websubhub-ballerina/hub_client.bal

```diff
@@ -157,37 +133,32 @@ isolated function retrieveContentType(string? contentType, string|xml|json|byte[
     }
 }
 
-# Retrieve signature for the content-distribution request payload.
-# 
-# + 'key - hashing key to be used (this is provided by the subscriber)
-# + payload - content-distribution request body
-# + return - `byte[]` containing the content signature or an `error` if there is any exception in the 
-#            function execution
-isolated function retrievePayloadSignature(string 'key, string|xml|json|byte[] payload) returns byte[]|error {
+isolated function retrievePayloadSignature(string contentType, string secret, string queryString, string|xml|json|byte[] payload) returns byte[]|error {
```

Comment:
Since `json` is a union of `()|boolean|int|float|decimal|string|json[]|map<json>`, we may be able to simplify `string|xml|json|byte[]` as `xml|json|byte[]`. I have observed this in other places of the module also. May be we can address this in a separate PR. WDYT?

Reply:
Yeah will update

Reply:
As discussed we will address this in a separate PR